doc/changes/changes_1.3.0.md

```diff
@@ -0,0 +1,29 @@
+# 1.3.0 - 2025-06-02
+
+## Summary
+
+With #441, please ensure that the location of the `version.py` is given for `Config.version_file`,
+which is specified in the `noxconfig.py`
+
+With #449, it's possible to customize what arguments are being using with `pyupgrade`
+via the `noxconfig.Config`:
+```python
+pyupgrade_args = ("--py310-plus",)
+```
```

Comment:
I assume, updating / upgrading dependencies can be hairy in some cases. This might have motivated allowing to customize the options. 

Would it make sense to add a section to the user guide?
In the first run, the section in the user guide it could be rather generic, maybe mentioning option `--py310-plus` and describe a potential reason when adding this option might be required. 
But later on, we could maybe use this section to add more scenarios and examples about how to upgrade?

Reply:
As discussed, a few things are missing :cry: , so I'd rather tackle it in a dedicated documentation task:
https://github.com/exasol/python-toolbox/issues/456